src/js/containers/search/filters/programSource/TASCheckboxTreeContainer.jsx

```diff
@@ -219,7 +220,7 @@ export default class TASCheckboxTree extends React.Component {
                                 <button
                                     className="shown-filter-button"
                                     value={label}
-                                    onClick={() => this.removeSelectedFilter(node)}
+                                    onClick={(e) => this.removeSelectedFilter(e, node)}
```

Comment:
Arrow functions in elements is a react anti-pattern due to the creation of a new function every time.
https://reactjs.org/docs/faq-functions.html

Reply:
Since this is an existing issue and I'm not introducing the issue can you approve? I am touching this file today with another PR for the tooltip and stuff so I can fix it in a follow up pr.